### PR TITLE
add CLING_MODULEMAP_PATH to CLHEP tool

### PR DESCRIPTION
**Pull request by davidlange6**
@vgvassilev identified this as missing from out last modules related changes.
@smuzaffar is this the right place to put the new envvar?

### CODE REVIEW

File: clhep-toolfile.spec [hunk 1]
```diff
@@ -30,6 +30,7 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/clhepheader.xml
   <runtime name="CLHEP_PARAM_PATH" value="$CLHEP_BASE"/>
   <runtime name="CMSSW_FWLITE_INCLUDE_PATH" value="$INCLUDE" type="path"/>
   <runtime name="ROOT_INCLUDE_PATH"  value="$INCLUDE" type="path"/>
+  <runtime name="CLING_MODULEMAP_PATH" value="$PCM_UTIL_BASE/include" type="path"/>
```
Comment (smuzaffar) [inline clhep-toolfile.spec]: why PCM_UTIL_BASE ?

**Comment by smuzaffar** [comment]
yes @davidlange6 this looks the right place

File: clhep-toolfile.spec (hunk 1 above)
Comment (smuzaffar) [reply clhep-toolfile.spec]: @davidlange6 , can you please fix this? You need to use $CLHEP_BASE instead of $PCM_UTIL_BASE :-)

Comment (davidlange6) [reply clhep-toolfile.spec]: whoops

**Comment by davidlange6** [comment]
fixed - I'll merge later if no one beats me to it.

**Comment by smuzaffar** [comment]
please test

**Pull request merged**